### DISCUSSION TITLE
Typora 使用教程

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#24] [2️⃣ 聊项目]
- https://zhuanlan.zhihu.com/p/293557841
- https://sspai.com/post/54912

<div align=center>
<img src="https://user-images.githubusercontent.com/15010826/161431030-a616ff00-5b2d-48a6-8d2b-97f35571ff51.png" width="300px">
</div>

**Comment by xgdyp** [comment]
https://github.com/xgdyp/Typora_Free_Download 

I fork a repo and this repo contains the last release of free Typora. Since Version1.0, Typora is no longer free and also has more stronger feats~

**Reply by will-ww** [reply]
> https://github.com/xgdyp/Typora_Free_Download
> 
> I fork a repo and this repo contains the last release of free Typora. Since Version1.0, Typora is no longer free and also has more stronger feats~

我入了一个付费版，确实好用~ :-)

**Comment by frank-zsy** [comment]
不太确定 Typora 的渲染是否有对应的前端库可以支持，其实本地写 Markdown 文档的工具挺多，而且很多都对 Markdown 语法做了很多扩展，使得本地使用非常方便。但当以网页形式发布或发布到 GitHub 仓库时就会使得大部分扩展语法无法被正确渲染，所以我一直比较克制的坚持使用 [GitHub Flavored Markdown](https://github.github.com/gfm/) 语法。

**Reply by will-ww** [reply]
> 不太确定 Typora 的渲染是否有对应的前端库可以支持，其实本地写 Markdown 文档的工具挺多，而且很多都对 Markdown 语法做了很多扩展，使得本地使用非常方便。但当以网页形式发布或发布到 GitHub 仓库时就会使得大部分扩展语法无法被正确渲染，所以我一直比较克制的坚持使用 [GitHub Flavored Markdown](https://github.github.com/gfm/) 语法。

是的，非常赞成用 [GitHub Flavored Markdown](https://github.github.com/gfm/) 的语法，基本图文的话，还是很不错的，然后也有一些流程图等扩展。

而且可以感受到，当 GitHub 这样的平台变成大家共同的平台后，GFM 甚至就成了一个标准~

**Reply by will-ww** [reply]
Typora 主要是它本地的用户体验，对于我这类文字型内容输出与格式控者，还是不错的~ :-)

**Reply by frank-zsy** [reply]
是的，看了一下 Typora 的上手文章，尤其在图床这块还是非常友好的，目前我自己的方式比较原生，体验没有这么好。
